libs/openchallenges/home/src/lib/home.component.html

```diff
@@ -1,4 +1,21 @@
 <main class="base mat-typography">
+  <div id="announcement">
+    <h4>
+      Welcome to
+      <a href="/about" target="_blank">OpenChallenges</a>!
```

Comment:
I made this suggestion earlier but could not retrieve it so sorry if you commented on it. I would remove the link here because the user may found absurd to be redirected to a website that are already on. I understand that you redirect to the About page, but we don't want them to go there before reading the home page.